modules/KIWILinuxRC.sh

```diff
@@ -6420,17 +6422,26 @@ function setupReadWrite {
                 FSTYPE=$FSTYPE_SAVE
             fi
             if [ "$RELOAD_IMAGE" = "yes" ] || \
-                ! mount -o ro $rwDevice $rwDir &>/dev/null
+                ! mount -o ro "$rwDevice" $rwDir &>/dev/null
             then
-                Echo "Creating filesystem for RW data on $rwDevice..."
-                if ! mkfs.ext3 -F $rwDevice >/dev/null;then
-                    Echo "Failed to create ext3 filesystem"
-                    return 1
+		Echo "Creating filesystem for RW data on $rwDevice..."
+                if [ "$kiwi_hybridpersistent" = "true" ];then
```

Comment:
> The indentation is broken here and in other places, could you fix that (4spaces) thanks

Yeah, I usually use 8 tabs instead of 4 spaces and when I figured that it must be breaking your style it was too late.